ayup/portfolio/src/main/webapp/style.css

```diff
@@ -0,0 +1,62 @@
+
+@import url('https://fonts.googleapis.com/css2?family=Playfair+Display+SC:ital,wght@0,400;0,700;0,900;1,400;1,700;1,900&display=swap');
+#content {
+  margin-left: auto;
+  margin-right: auto;
+  background-color: red;
+}
+
+#greeting-container {
+  margin-top: 20px;
+}
+.jumbotron-image {
+    max-height:100px;
+}
+.display-4 {
+  font-family: 'Playfair Display SC', serif;
+  font-weight: 400 !important;
```

Comment:
It's generally best practice to try to figure out what's going rather than use an !important tag: https://uxengineer.com/css-specificity-avoid-important-css/

Reply:
Thanks, I'll see what I can do to remove the !important.

Reply:
We had a host meeting.. no need to spend any more time on this if you haven't figured it out. : /